index.rst

```diff
@@ -217,7 +218,6 @@ Output Components
     TLC59208F, components/output/tlc59208f, tlc59208f.jpg
     MY9231/MY9291, components/output/my9231, my9231.svg
     SM16716, components/output/sm16716, sm16716.svg
-    MCP4725, components/output/mcp4725, mcp4725.jpg
```

Comment:
Please check your commits to not remove unrelated lines

Reply:
update ok